packages/jest-cli/src/runJest.js

```diff
@@ -151,7 +152,7 @@ const runJest = async (
   pipe: stream$Writable | tty$WriteStream,
   testWatcher: TestWatcher,
   startRun: () => *,
-  onComplete: (testResults: any) => any,
+  onComplete: (testResults: AggregatedResult) => any,
```

Comment:
that does seem like a correct type in this situation, although it sucks that we have to weaken it.
I don't mind having it weak if @cpojer doesn't mind.
We can always refactor and strengthen the types later if we need to